src/main/java/org/opensearch/neuralsearch/processor/AgenticQueryTranslatorProcessor.java

```diff
@@ -59,6 +61,7 @@ public void processRequestAsync(
         PipelineProcessingContext requestContext,
         ActionListener<SearchRequest> requestListener
     ) {
+        EventStatsManager.increment(EventStatName.AGENTIC_QUERY_TRANSLATOR_PROCESSOR_EXECUTIONS);
```

Comment:
Any need for an info stat to count number of query translator processors in pipelines, even when queries aren't being run? Like https://github.com/opensearch-project/neural-search/blob/4bc12475479d8242982671ca8ceb3dddf9b175cf/src/main/java/org/opensearch/neuralsearch/stats/info/InfoStatName.java#L30

Reply:
Sure, raised https://github.com/opensearch-project/neural-search/pull/1530